dll/shellext/stobject/lang/de-DE.rc

```diff
@@ -16,7 +16,7 @@ STRINGTABLE
 BEGIN
     //Power related strings
     IDS_PWR_PROPERTIES        "&Energieverwaltungseigenschaften einstellen"
-    IDS_PWR_METER             "Batterieanzeige ö&ffnen"
+    IDS_PWR_METER             "Batterieanzeige öffnen"
```

Comment:
It's not a typo. Did you tried to actually run ReactOS and check this string there?

Reply:
Well no. If I was wrong I apologise.

Reply:
No problem, if you did not tested, these ampersands are for keyboard shortcuts:

![image](https://user-images.githubusercontent.com/578406/90344476-e55feb80-e022-11ea-88d0-eff0d07037e3.png)

You still can add commits with actual translations to this Pull Request. :wink: